deploy/olm-catalog/ibm-iam-operator/3.17.0/ibm-iam-operator.v3.17.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,2045 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
```

Comment:
Change the respective image tags for alm-examples

Reply:
Done. Please check